src/Symfony/Component/Process/ProcessBuilder.php

```diff
@@ -226,6 +226,20 @@ public function setOption($name, $value)
     }
 
     /**
+     * Sets proc_open options.
+     *
+     * @param array $options Key value pair of options
+     *
+     * @return ProcessBuilder
+     */
+    public function setOptions(array $options)
```

Comment:
It is quite conflicting with this one [ProcessBuilder#L221](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Process/ProcessBuilder.php#L221)

`setOption` is adding a new option in options array if I add `addOptions` it would result in confusion.

Are you sure we should still name it to `addOptions`?  Or we can create two methods `setOptions` will overwrite and `addOptions` will add it into current options array.